package/MDAnalysis/auxiliary/core.py

```diff
@@ -56,8 +62,13 @@ def get_auxreader_for(auxdata, format=None):
         else:
             ## arrays etc
             pass
-    format = format.upper()
-    try:
-        return _AUXREADERS[format]
-    except KeyError:
-        raise ValueError("Unknown auxiliary data format for {0}".format(auxdata))
+        format = format.upper()
+        try:
+            return _AUXREADERS[format]
+        except KeyError:
+            raise ValueError("Unknown auxiliary data format for {0}".format(auxdata))
+    else:
+        try:
+            return _AUXREADERS[format]
```

Comment:
Hm, I hadn't really thought beyond the XVGReader; it might depend how the guessing-format-of-none-file-data works, but I image having them all be strings (to be consistent) would make sense?